src/herder/TxQueueLimiter.cpp

```diff
@@ -103,11 +103,14 @@ TxQueueLimiter::canAddTx(TransactionFrameBasePtr& newTx,
     auto txBaseFee = newTx->getBaseFeeBid();
     if (txBaseFee <= getMinBaseFee())
     {
-        return false;
+        auto minFee = newTx->getFee(Config::CURRENT_LEDGER_PROTOCOL_VERSION,
```

Comment:
Nitpick: The formatting on this is bizarre. Did `clang-format` force this?

Reply:
deleted